src/main/scala/com/gu/mobile/notifications/football/Streams.scala

```diff
@@ -26,8 +27,10 @@ trait MatchDayStream extends Logging {
       val todaysMatchesFuture = PaMatchDayClient(PaFootballClient).today
 
       todaysMatchesFuture onFailure {
-        case error =>
-          logger.error("Error getting today's matches from PA", error)
+        case pae: PaClientErrorsException if pae.msg.contains("No data available") =>
+          logger.info(s"No match data available for today [${LocalDate.now}]")
+        case error: Throwable =>
+          logger.error(s"Error getting today's matches from PA [${error.getMessage}]")
```

Comment:
That would be nice to catch exceptions rather than throwable (will catch out of memory, or class not found, which you don't want)
Also, you could trace the exception with the error
`logger.error(s"Error getting today's matches from PA [${error.getMessage}]", e)`